easybuild/easyconfigs/a/ANSYS/ANSYS-21.1.eb

```diff
@@ -0,0 +1,49 @@
+name = 'ANSYS'
+version = '21.1'
+
+homepage = 'http://www.ansys.com'
+description = """
+ANSYS simulation software enables organizations to confidently predict
+how their products will operate in the real world. We believe that every product
+is a promise of something greater.
+
+Typical Usage to check license server access:
+   module load tools/ANSYS
+   export LM_LICENSE_FILE=$(cat $EBROOTANSYS/shared_files/licensing/ansyslmd.ini | head -n 1 | cut -d = -f 2)
+   $EBROOTANSYS/shared_files/licensing/linx64/lmutil lmstat -a
```

Comment:
probably better suited to `usage` or `modloadmsg`

Reply:
Indeed ;) done in [b5058b](https://github.com/easybuilders/easybuild-easyconfigs/pull/15251/commits/2b5058b073efd5bae57b8a055d2f8e6505406d99)